services/shared/src/main/java/com/wedriveu/services/shared/entity/VehicleCounter.java

```diff
@@ -0,0 +1,83 @@
+package com.wedriveu.services.shared.entity;
+
+/**
+ * @author Stefano Bernagozzi
+ */
+
+public class VehicleCounter {
+    private int available;
+    private int broken;
+    private int booked;
+    private int recharging;
+    private int stolen;
+
+
+    public VehicleCounter() {
+        available = 0;
```

Comment:
Is this initialization needed? The default value for `int` primitive data type is not 0?